rollup.config.js

```diff
@@ -33,6 +33,7 @@ const commonPlugins = [
   replace({
     'process.env.PACKAGE_NAME': JSON.stringify(process.env.PACKAGE_NAME),
     'process.env.PACKAGE_VERSION': JSON.stringify(process.env.PACKAGE_VERSION),
+    'process.env.SKIP_VERSION_CHECK': '0',
```

Comment:
I think we should just use real variable names like `AUTH_JS` does. I don't see a reason they need to be on the `process.env` object